src/components/organisms/news/news.scss

```
@@ -0,0 +1,37 @@
+@use "../../variables" as *;
+@use "../../molecules/news-and-events-more/news-and-events-more" as *;
+@use "../../molecules/news-and-events-title/news-and-events-title.scss" as *;
+
```

```suggestion
.news {
  display: flex;
  flex-direction: column;
```
「お知らせ一覧」を左に寄せるためにalign-self: flex-end;を使用するため必要だと思います
align-selfについては検索すればでると思うので、詳しく知りたい場合検索してみてください。